.github/workflows/deploy.yml

```diff
@@ -53,7 +53,7 @@ jobs:
         if: github.ref == 'refs/heads/development'
         env:
           SITE_URL: https://covid-staging.citybureau.org
-          S3_BUCKET: city-bureau-covid-staging
+          S3_BUCKET: site-owner-covid-staging
```

Comment:
these I changed as part of an overzealous find and replace. i haven't touched the deploy stuff yet, since its set up specific to S3 and github actions. 

I did successfully deploy a version of the site to netlify, it might be worth making it easier to deploy the project in case organizations don't have an s3 setup. For those that do, there's plenty here that will take them far which is great!